DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: apLCMS
 Title: Adaptive processing of LC-MS data
-Version: 6.6.8
+Version: 6.7.0
 Date: 2019-11-20
 Author: Tianwei Yu <tianwei.Yu@emory.edu>
 Maintainer: Tianwei Yu <tianwei.Yu@emory.edu>
```

Comment:
I am glad you've noticed this. I already had my thoughts on this but as I see that you would lean in the same direction as me I will encourage those. Thus, I propose the following changes:

1) Name the R package recetox-aplcms in order to be consistent with conda.
2) Restart versioning, starting from 1.0.0
3) Add us to the auothors (I would prefer to add the whole group as an GitHub recetox organisation but we don't have any group email).
4) Since recetox-aplcms is OUR package, we replace Tianwei Yu with us.
5) When doing PR to the upstream we need to change the R package name and the other metadata to the original.

Do you agree?

Reply:
Bad news on the package naming:

> There are three formal requirements:
> 
> The name can only consist of letters, numbers, and periods, i.e., ..
> It must start with a letter.
> It cannot end with a period.
> 
> Unfortunately, this means you can’t use either hyphens or underscores, i.e., - or _, in your package name. We recommend against using periods in package names, due to confusing associations with file extensions and S3 methods.

So that leaves us with the name `recetoxaplcms`. What do you think? I think it's ugly but it will do just fine.

Reply:
Alternatively `recetox.aplcms` against their recommendations (but allowed)

Reply:
I'd stick with the recetox.aplcms. I wouldn't dare removing Tianwei Yu from the list of authors, since indirectly, he is also author of this package. But certainly not a maintainer.

Reply:
I adapted this in https://github.com/RECETOX/recetox-aplcms/pull/14/commits/18fa85e53777e555a1243b509af84d462dffc21c sticking with Jirka as the contact point until we get some common email we can use. Hope you don't min @xtracko